recipes/museotoolbox/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "museotoolbox" %}
+{% set version = "0.12" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 30201a51d6033c1b191bd822741f7cbf073053926928d818ea874bfcad3f9917
+
+build:
+  noarch: python
```

Comment:
noarch packages don't need to run on Appveyor, they only need to pass on linux

Reply:
@scopatz, is there anything I can do to avoid running on Appveyor? I looked at some released recipes with noarch and don't found how to avoid appveyor from running. 
Thank you 😊

Reply:
Not on staged-recipes, sadly.  But it doesn't need tp pass